config/version.php

```diff
@@ -6,5 +6,5 @@
   'prerelease_version' => '',
   'hash_version' => 'gfb857ccf5',
   'full_hash' => 'v7.1.16-510-gfb857ccf5',
-  'branch' => 'master',
+  'branch' => 'develop',
```

Comment:
Weird, wonder how this happened? Well, this is the way it's supposed to be anyways.

Reply:
Weird merge from the PR before this one. I think the PR before this one came from master.